reactor-core/src/jcstress/java/reactor/core/publisher/SinkManySerializedStressTest.java

```diff
@@ -69,7 +69,7 @@ public void arbiter(LLI_Result r) {
 	@Outcome(id = {"FAIL_TERMINATED, OK, 1"}, expect = ACCEPTABLE, desc = "onNext after onComplete")
 	@Outcome(id = {"OK, OK, 1"}, expect = ACCEPTABLE, desc = "onComplete after onNext")
 	@State
-	public static class TerminatedVsOnNextStressTest extends SerializedManySinkStressTest {
+	public static class TerminatedVsOnNextManySerializedStressTest extends SinkManySerializedStressTest {
```

Comment:
same as in above @bsideup comment: rename back to `TerminatedVsOnNextStressTest`